### PR TITLE
sprint 9 task#60 Fix bug not vote when has image

### PR DESCRIPTION
**Pull request by nhahv0902**

### CODE REVIEW

File: app/src/main/java/com/framgia/fpoll/ui/votemanager/updateoption/UpdateOptionContract.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+public interface UpdateOptionContract {
+    interface View extends BaseView {
+
+        void updateOption(Option option);
+
+        void showPickDate(Option option);
+
+        void pickImageFromGallery(Option option);
+
+        void showMessage(int message);
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/fpoll/ui/votemanager/updateoption/UpdateOptionContract.java]: pls format this code

File: app/src/main/java/com/framgia/fpoll/ui/votemanager/updateoption/UpdateOptionFragment.java [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+import static com.framgia.fpoll.util.Constant.BundleConstant.BUNDLE_POLL_ITEM;
+import static com.framgia.fpoll.util.Constant.RequestCode.IMAGE_PICKER_SELECT;
+import static com.framgia.fpoll.util.Constant.RequestCode.PERMISSIONS_REQUEST_WRITE_EXTERNAL;
+import static com.framgia.fpoll.util.Constant.Tag.DATE_PICKER_TAG;
+import static com.framgia.fpoll.util.Constant.Tag.TIME_PICKER_TAG;
+
+public class UpdateOptionFragment extends DialogFragment
+        implements UpdateOptionContract.View, DatePickerDialog.OnDateSetListener,
+        TimePickerDialog.OnTimeSetListener {
+    private FragmentUpdateOptionBinding mBinding;
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/fpoll/ui/votemanager/updateoption/UpdateOptionFragment.java]: change this variable to local variable

File: app/src/main/java/com/framgia/fpoll/ui/votemanager/updateoption/UpdateOptionFragment.java [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+            Intent intent = new Intent(Intent.ACTION_PICK,
+                    android.provider.MediaStore.Images.Media.EXTERNAL_CONTENT_URI);
+            startActivityForResult(intent, IMAGE_PICKER_SELECT);
+        }
+    }
+
+    @Override
+    public void showPickDate(Option option) {
+        if (mCalendar == null) mCalendar = Calendar.getInstance();
+        DatePickerDialog dpd = DatePickerDialog.newInstance(this, mCalendar.get(Calendar.YEAR),
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/fpoll/ui/votemanager/updateoption/UpdateOptionFragment.java]: pls rename this variable

File: app/src/main/java/com/framgia/fpoll/ui/votemanager/updateoption/UpdateOptionFragment.java [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+        if (requestCode == IMAGE_PICKER_SELECT
+                && resultCode == RESULT_OK
+                && data != null
+                && mOption != null) {
+            Uri selectedImage = data.getData();
+            String[] filePathColumn = { MediaStore.Images.Media.DATA };
+            Cursor cursor = getActivity().getContentResolver()
+                    .query(selectedImage, filePathColumn, null, null, null);
+            if (cursor == null) return;
+            cursor.moveToFirst();
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/fpoll/ui/votemanager/updateoption/UpdateOptionFragment.java]: pls check cursor.moveToFirst value.

File: app/src/main/java/com/framgia/fpoll/ui/votemanager/updateoption/UpdateOptionContract.java (hunk 1 above)
Comment (nhahv0902) [inline app/src/main/java/com/framgia/fpoll/ui/votemanager/updateoption/UpdateOptionContract.java]: fixed

**Pull request merged**